PIPS/pip-009.md

```diff
@@ -0,0 +1,58 @@
+    PIP: 009
+    Title: Investor Referral Reward Program (aka Hustler PIP)
+    Author: <KRostislav Grubi>
+    Created: <2018-03-18>
+    Requires: PIP002
+
+
+## Simple Summary
+Bring in new customers (investments) to the project and receive your cut as a reward. That can help us stay independent. 
+
+## Motivation
+Project needs money to sustain itself and stay independent. This PIP aims to incentivise people to look for a way to bring more customers (investments) to the PARSEC Labs project.
```

Comment:
I think customers and investments are not related. Why are they mentioned next to each other?